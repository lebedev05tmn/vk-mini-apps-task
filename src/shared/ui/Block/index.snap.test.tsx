import { Block } from "./index";

const ReactTestRenderer = require("react-test-renderer");

describe("Block render", () => {
  test("Should Block with children", () => {
    const tree = ReactTestRenderer.create(
      <Block>
        <div>hello world</div>
      </Block>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
