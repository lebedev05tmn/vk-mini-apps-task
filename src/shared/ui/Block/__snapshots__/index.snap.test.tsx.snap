// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Block render Should Block with children 1`] = `
<div
  className="vkuiSplitCol vkuiSplitCol--viewWidth-none vkuiInternalSplitCol--viewWidth-none"
  style={
    {
      "maxWidth": undefined,
      "minWidth": undefined,
      "width": undefined,
    }
  }
>
  <section
    className="vkuiInternalGroup vkuiGroup vkuiGroup--sizeX-none vkuiInternalGroup--sizeX-none vkuiGroup--mode-none vkuiInternalGroup--mode-none vkuiGroup--padding-m"
  >
    <div>
      hello world
    </div>
  </section>
  <div
    className="vkuiSpacing vkuiGroup__separator vkuiGroup__separator--spacing"
    style={
      {
        "height": 16,
        "padding": "8px 0",
      }
    }
  />
  <div
    className="vkuiSeparator vkuiSeparator--padded vkuiGroup__separator vkuiGroup__separator--separator"
  >
    <hr
      className="vkuiSeparator__in"
    />
  </div>
</div>
`;
